src/OpenTelemetry.Exporter.Console/ConsoleLogRecordExporter.cs

```diff
@@ -82,21 +82,28 @@ public override ExportResult Export(in Batch<LogRecord> batch)
                     this.WriteLine($"{"LogRecord.FormattedMessage:",-RightPaddingLength}{logRecord.FormattedMessage}");
                 }
 
+                if (logRecord.Body != null)
+                {
+                    this.WriteLine($"{"LogRecord.Body:",-RightPaddingLength}{logRecord.Body}");
+                }
+
+#pragma warning disable CS0618 // Type or member is obsolete
                 if (logRecord.State != null)
```

Comment:
Would it be wrong to remove this from the console exporter? Declare it a bug due to the potential risk of accessing outside the thread that wrote the log?

I guess a batch processor with the console exporter may be unlikely...

Reply:
Updated. She gone.